third_party/gfootball_engine/CMakeLists.txt

```diff
@@ -129,10 +130,10 @@ add_library(gamelib ${GAME_SOURCES} ${GAME_HEADERS})
 add_library(menulib ${MENU_SOURCES} ${MENU_HEADERS})
 add_library(datalib ${DATA_SOURCES} ${DATA_HEADERS})
 
-set(LIBRARIES gamelib menulib datalib blunted2 rt
-   ${Boost_LIBRARIES} ${PYTHON_LIBRARY}
+set(LIBRARIES gamelib menulib datalib blunted2
+   Boost::filesystem Boost::system Boost::thread Boost::${BOOST_PYTHON_VERSION} ${PYTHON_LIBRARY}
```

Comment:
Linking to ${Boost_LIBRARIES} is not sufficient (see [this link](https://cmake.org/cmake/help/v3.8/module/FindBoost.html)) so components have to be explicitly linked.